pkgs/development/r-modules/wrapper.nix

```diff
@@ -1,9 +1,13 @@
-{ stdenv, R, makeWrapper, recommendedPackages, packages }:
+{ stdenv, R, libcxx, makeWrapper, recommendedPackages, packages }:
 
 stdenv.mkDerivation {
   name = R.name + "-wrapper";
 
-  buildInputs = [makeWrapper R] ++ recommendedPackages ++ packages;
+  buildInputs = [makeWrapper R] ++ recommendedPackages ++ packages
+    ++ stdenv.lib.optionals stdenv.isDarwin R.darwinFrameworks;
+
+  NIX_CFLAGS_COMPILE =
+    stdenv.lib.optionalString stdenv.isDarwin "-I${libcxx}/include/c++/v1";
```

Comment:
I can only try it out tomorrow and see if the packages I needed compiled without that.

Peter Simons wrote:
> In pkgs/development/r-modules/wrapper.nix 
> <https://github.com/NixOS/nixpkgs/pull/10623#discussion_r49006746>:
>
> >
> >   stdenv.mkDerivation {
> >     name = R.name +"-wrapper";
> >
> >  -  buildInputs = [makeWrapper R] ++ recommendedPackages ++ packages;
> >  +  buildInputs = [makeWrapper R] ++ recommendedPackages ++ packages
> >  +    ++ stdenv.lib.optionals stdenv.isDarwin R.darwinFrameworks;
> >  +
> >  +  NIX_CFLAGS_COMPILE =
> >  +    stdenv.lib.optionalString stdenv.isDarwin"-I${libcxx}/include/c++/v1";
>
> Are you certain that those changes in the wrapper are necessary? Based 
> on my understanding, the modifications to |buildInputs| and 
> |NIX_CFLAGS_COMPILE| have no effect on the wrapper whatsoever, so I 
> tend to believe that those changes can be just dropped.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/NixOS/nixpkgs/pull/10623/files#r49006746>.
>